docs/releases/1.11.3.txt

```diff
@@ -6,6 +6,11 @@ Django 1.11.3 release notes
 
 Django 1.11.3 fixes several bugs in 1.11.2.
 
+Minor feature
```

Comment:
This won't be backported to 1.11 per our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions). This should be in the 2.0 release notes.